social-proof-section-master/styles.css

```
@@ -0,0 +1,275 @@
+html {
+  height: 100%;
+}
+
+body {
+  background-color: hsl(0, 0%, 100%);
+  color: hsl(300, 43%, 22%);
+  font-size: 15px;
+  font-family: "League Spartan";
+  font-weight: 400;
+  height: 100%;
+  line-height: 1.2;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 3em;
+  line-height: 1;
+}
+
+p {
+  line-height: 1.5;
+  font-weight: 500;
+}
+
+#bg-top {
+  background: no-repeat url("./assets/images/bg-pattern-top-desktop.svg");
+  width: 100%;
+  height: 100%;
+  background-size: 35%;
+  background-position: top left;
+  position: fixed;
+  z-index: -1000;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+#bg-bottom {
+  background: no-repeat url("./assets/images/bg-pattern-bottom-desktop.svg");
+  width: 100%;
+  height: 100%;
+  background-size: 65%;
+  background-position: bottom right;
+  position: fixed;
+  z-index: -1000;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+#container {
+  height: 100%;
+  display: flex;
+}
+
+#content-container {
+  margin: auto;
+  max-width: 80%;
+}
+
+#header-container {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 20px;
+  align-items: center;
+}
+
+#info-container {
+  width: 50%;
+}
+
+#info-container h1 {
+  max-width: 60%;
+}
+
+#info-container p {
+  max-width: 70%;
+  line-height: 1.4;
+  color: hsl(303, 10%, 53%);
+}
+
+#rating-section-container {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.rating-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  background-color: hsl(300, 24%, 96%);
+  border-radius: 8px;
+  align-items: center;
+  font-weight: 700;
+  max-width: 80%;
+  margin: 0 0 12px 0;
+}
+
+.rating-container p {
+  font-weight: 700;
+}
+
+.shift-right-small {
+  -webkit-transform: translateX(10%);
+  -ms-transform: translateX(10%);
+  transform: translateX(10%);
+}
+
+.shift-right-medium {
+  -webkit-transform: translateX(20%);
+  -ms-transform: translateX(20%);
+  transform: translateX(20%);
+}
+
+#testimonials-section-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.testimonials-container {
+  color: hsl(0, 0%, 100%);
+}
+
+.testimonial {
+  border-radius: 8px;
+  padding: 25px;
+  background-color: hsl(300, 43%, 22%);
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+.author-section {
+  display: flex;
+  flex-direction: row;
+}
+
+.author-section img {
+  width: 100%;
+  height: auto;
+  border-radius: 50%;
+}
+
+.avatar-image {
+  max-height: 32px;
+  max-width: 32px;
+  padding-right: 12px;
+  margin-right: 10px;
+}
+
+.author-identity {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+.pink {
+  color: hsl(333, 80%, 67%);
+}
+
+.shift-margin-top-20 {
+  margin-top: 20px;
+}
+
+.shift-margin-top-40 {
+  margin-top: 40px;
+}
+
+@font-face {
+  font-family: "League Spartan";
+  src: local("./assets/static/LeagueSpartan-VariableFont_wght.ttf");
```

When I have opened the exercise locally the font did not load. I think I know why:
- `local` is for using the fonts that are installed on your machine
- you need to use `url` here:
```suggestion
  src: url("./assets/static/LeagueSpartan-VariableFont_wght.ttf");
```